tck/features/clauses/create/Create2.feature

```diff
@@ -0,0 +1,296 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Create2 - Creating relationships
+
+  Scenario: [1] Creating two nodes and a single relationship in a single pattern
+    Given any graph
+    When executing query:
+      """
+      CREATE ()-[:R]->()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+
+  Scenario: [2] Creating two nodes and a single relationship in separate patterns
+    Given any graph
+    When executing query:
+      """
+      CREATE (a), (b),
+             (a)-[:R]->(b)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+
+  Scenario: [3] Creating two nodes and a single relationship in separate clauses
+    Given any graph
+    When executing query:
+      """
+      CREATE (a)
+      CREATE (b)
+      CREATE (a)-[:R]->(b)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+
+  Scenario: [4] Creating two nodes and a single relationship in reversed direction
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (:A)<-[:R]-(:B)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +labels        | 2 |
+    When executing control query:
+      """
+      MATCH (a:A)<-[:R]-(b:B)
+      RETURN a, b
+      """
+    Then the result should be, in any order:
+      | a    | b    |
+      | (:A) | (:B) |
+
+  Scenario: [5] Creating a single relationship between two existing nodes
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:X)
+      CREATE (:Y)
+      """
+    When executing query:
+      """
+      MATCH (x:X), (y:Y)
+      CREATE (x)-[:R]->(y)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +relationships | 1 |
+
+  Scenario: [6] Creating a single relationship between two existing nodes in reversed direction
```

Comment:
reversed direction -> the reverse direction

Reply:
done